### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ndrinta**
Just fixed some typos :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,28 +30,28 @@ The changes from the main branch include:
 * Corrected unique_id for BMS Cell mV low
 * Added automation to upload generation and consumption data to pvoutput.org
 
-Access to your inverter data can be acheived by connecting the inverter's RS485 modbus to an RS485 to USB adapter or RS485 to WIFI/LAN adapter. Note: this requires basic electronics competencies to connect 2 wires to the inverters CT / COM connector.
+Access to your inverter data can be achieved by connecting the inverter's RS485 Modbus to an RS485 to USB adapter or RS485 to WiFi/LAN adapter. Note: this requires basic electronics competencies to connect 2 wires to the inverter's CT / COM connector.
 
 ---
-** Please note: April 2023, @nathanmarlor has mirrored and extended functionality in this alternative https://github.com/nathanmarlor/foxess_modbus - you may wish to look at that as well!
+** Please note: April 2023, @nathanmarlor has mirrored and extended functionality in this alternative https://github.com/nathanmarlor/foxess_Modbus - you may wish to look at that as well!
```
Comment (TonyM1958) [inline README.md:36]: don't think you can change the name of a file as they are case sensitive

File: README.md [hunk 2]
```diff
@@ -30,28 +30,28 @@ The changes from the main branch include:
...
 
 ---
 
 
 ## RS485 connection to H1, AC, AIO, H1-G2, AC-G2 and KH series inverters (recommended)
-* Hardware configuration instructions can be found on Nathan's [wiki](https://github.com/nathanmarlor/foxess_modbus/wiki)
+* Hardware configuration instructions can be found on Nathan's [wiki](https://github.com/nathanmarlor/foxess_Modbus/wiki)
```
Comment (TonyM1958) [inline README.md:42]: change if case in a link

File: README.md [hunk 3]
```diff
@@ -30,28 +30,28 @@ The changes from the main branch include:
...
 * Connect RS485A to pin 4 and RS485B to pin 3 of the Meter/CT/RS485 connector using a suitable length of UTP cable (e.g. network cable)
 * Go to the inverter front panel Settings, Communication, RS485, Device ID and check the slave ID for the inverter is set to 247
-* Use modbusXX_RS485.yaml when using an RS485 to USB adapter
-* Use modbusXX_RS485_LAN.yaml when using an RS485 to Wifi/LAN adapter
+* Use ModbusXX_RS485.yaml when using an RS485 to USB adapter
+* Use ModbusXX_RS485_LAN.yaml when using an RS485 to Wifi/LAN adapter
```
Comment (TonyM1958) [inline README.md:46]: change of case referring to file names

File: README.md [hunk 4]
```diff
@@ -30,28 +30,28 @@ The changes from the main branch include:
...
 * where XX is H1 for H1, AC1 or AIO series, H1G2 for H1-G2 or AC-G2 series or KH for K series
 
 ## RS485 connection to H3 or AC3 series inverter
-* Hardware configuration instructions for connection to RS485 can be found on the [wiki](https://github.com/nathanmarlor/foxess_modbus/wiki)
+* Hardware configuration instructions for connection to RS485 can be found on the [wiki](https://github.com/nathanmarlor/foxess_Modbus/wiki)
```
Comment (TonyM1958) [inline README.md:50]: change of case in a link

File: README.md [hunk 5]
```diff
@@ -30,28 +30,28 @@ The changes from the main branch include:
...
 * Connect RS485A to pin 1 and RS485B to pin 2 of the Meter/RS485 connector using a suitable length of UTP cable (e.g. network cable)
 * Go to inverter front panel, Settings, Communication, RS485, Device ID and set the slave ID for the inverter to 247 (the default is 000)
-* Use modbusH3_RS485.yaml when using an RS485 to USB adapter
-* Use modbusH3_RS485_LAN.yaml when using an RS485 to Wifi/LAN adapter
+* Use ModbusH3_RS485.yaml when using an RS485 to USB adapter
+* Use ModbusH3_RS485_LAN.yaml when using an RS485 to Wifi/LAN adapter
```
Comment (TonyM1958) [inline README.md:54]: change of case in a filename

File: README.md [hunk 6]
```diff
@@ -61,26 +61,26 @@ Access to your inverter data can be acheived by connecting the inverter's RS485
 
 ## Change Log
 v1.4.9:<br>
-Update H3 registers by cross-reference to other inverter models. Addresses to be confirmed are marked tbc in 'modbusH3_RS485_LAN.yaml'
+Update H3 registers by cross-reference to other inverter models. Addresses to be confirmed are marked tbc in 'ModbusH3_RS485_LAN.yaml'
```
Comment (TonyM1958) [inline README.md:64]: change of case in filename

File: README.md [hunk 7]
```diff
@@ -61,26 +61,26 @@ Access to your inverter data can be acheived by connecting the inverter's RS485
...
 
 v1.4.8:<br>
 Fix errors in 'template.yaml' and 'templateH1G2.yaml' affecting Battery Warranty Remaining and Battery Life Remaining.<br>
-Improve handling of unknonw values in 'template.yaml' and 'templateH1G2.yaml'.<br>
+Improve handling of unknown values in 'template.yaml' and 'templateH1G2.yaml'.<br>
 Setting for the number of days to keep moved to 'secrets.yaml'. See [note](https://github.com/TonyM1958/HA-FoxESS-Modbus/issues/20) for info on configuring this.
 
 v1.4.6:<br>
-Updates to 'modbusH1G2_RS485..yaml', 'templateH1G2.yaml' and 'viewsH1G2_sensor.yaml'.
+Updates to 'ModbusH1G2_RS485..yaml', 'templateH1G2.yaml' and 'viewsH1G2_sensor.yaml'.
```
Comment (TonyM1958) [inline README.md:72]: change of case in a filename

File: README.md [hunk 8]
```diff
@@ -61,26 +61,26 @@ Access to your inverter data can be acheived by connecting the inverter's RS485
...
+Moved preset values to input helpers to avoid overwriting values during HACS update. This requires some additional settings to be added to configuration.yaml. See 'template_configuration.yaml'.<br>
 See [note](https://github.com/TonyM1958/HA-FoxESS-Modbus/issues/19) for info on configuring this.
 
 v1.4.3:<br>
 Moved database settings to recorder_ha.yaml and recorder_maria.yaml so the database being used for history does not change during HACS update.<br>
 See [note](https://github.com/TonyM1958/HA-FoxESS-Modbus/issues/18) if you get a configuration error after updating.
 
 v1.4.2:<br>
-Added 'modbusH1G2_RS485.yaml' and 'modbusH1G2_RS485_LAN.yaml' to support H1-G2 and AC-G2 series inverters.<br>
+Added 'ModbusH1G2_RS485.yaml' and 'ModbusH1G2_RS485_LAN.yaml' to support H1-G2 and AC-G2 series inverters.<br>
```
Comment (TonyM1958) [inline README.md:86]: change of case in a filename

**Review by TonyM1958 — CHANGES_REQUESTED** [review]
Thanks - I've added review comments. The main thing is that as link and filenames are case sensitive, I don't think you can change the case of these - mainly modbus to Modbus.

File: README.md (hunk 1 above)
Comment (Ndrinta) [reply README.md:36]: Oh! You're so right... Might even work (in links at least) but that's not the point, I shouldn't have done a copy and replace "blindly". Will fix

File: README.md (hunk 7 above)
Comment (Ndrinta) [reply README.md]: Also fixing double-dot that I just noticed.

File: README.md [hunk 9]
```diff
@@ -9,17 +9,17 @@ This web site, code and contents are not supported by or endorsed by Fox ESS. An
    </br>
 </h2>
 
-This site contains code and information for integrating Fox ESS inverters with Home Assistant. It is a refactored and evolved version, derrived from https://github.com/StealthChesnut/HA-FoxESS-Modbus.
+This site contains code and information for integrating Fox ESS inverters with Home Assistant. It is a refactored and evolved version, derived from https://github.com/StealthChesnut/HA-FoxESS-Modbus.
 
 The changes from the main branch include:
 
-* Simplified configuration: fewer lines are required in your configuration.yaml with modbus, sensors, templates and utility_meter integrations split into separate include files
+* Simplified configuration: fewer lines are required in your configuration.yaml with Modbus, sensors, templates and utility_meter integrations split into separate include files
```
Comment (TonyM1958) [inline README.md:16]: Sorry, missed this one - these are literals, probably should be quited. 'modbus' is the name of the included section in configuration.yaml, not 'Modbus'.

**Review by TonyM1958 — CHANGES_REQUESTED** [review]
Sorry, one more, missed this first time.

**Pull request merged**